criu/cgroup.c

```diff
@@ -1064,27 +1069,34 @@ bool is_special_property(const char *prop)
 	return false;
 }
 
-static int userns_move(void *arg, int fd, pid_t pid)
+static int userns_move(void *args, int fd, pid_t pid)
 {
 	char pidbuf[32];
 	int cg, len, err;
+	pid_t move_id;
+	struct task_id_path *args_path = (struct task_id_path*) args;
 
-	len = snprintf(pidbuf, sizeof(pidbuf), "%d", pid);
+	if (args_path->id != pid)
+		move_id = args_path->id;
+	else
+		move_id = pid;
```

Comment:
So you use `args_path->id` if those two are not the same but `pid` if they are the same. I think it can be simplified.